### ISSUE TITLE
elytras

### ISSUE THREAD

**Issue by MaftPLays** [cabaletta/baritone#2328]
my curent goal is to reach the world border in survival, and I wanted to use my elytra to reach it, but I did not find any command that allowed baritone to use elytras.

It could be simple like, allowElytra and baritone goes flying in the sky straight to heaven.

It is self explanatory, you could fly out and around with the elytra in minecraft while, in real life, sleeping or something..

I don't have any experience with programing so a may be acting a little like a dumbass.
Sorry for any grammar mistakes, I tried to make this readable the best I could but english is not my main language so cut me some slack.

**Comment by scorbett123** [comment]
This is a duplicate of multiple other issues.

**Comment by CesiumCs** [comment]
[ElytraBot](https://github.com/bebeli555/ElytraBot)

**Comment by l1ving** [comment]
you'd think, in 2021 people would begin to realize they don't have the most original ideas ever
